core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -98,10 +101,17 @@ object ConsumerPerformance {
       consumerConnector.shutdown()
     }
     val elapsedSecs = (endMs - startMs) / 1000.0
```

Comment:
I think the main point of this patch is to compute the throughput statistics based only on the time spent fetching, so maybe we can move this below and just use `fetchTimeInMs.get / 1000.0`? We could also report `MB.sec` as two separate values: `total.MB.sec` which uses the total time and `fetch.MB.sec` which uses only the fetch time. Similarly for `nMsg.sec`.